contracts/optimistic-ethereum/OVM/execution/OVM_ExecutionManager.sol

```diff
@@ -1806,7 +1808,6 @@ contract OVM_ExecutionManager is iOVM_ExecutionManager, Lib_AddressResolver {
         messageContext.isStatic = false;
 
         messageRecord.nuisanceGasLeft = 0;
-        messageRecord.revertFlag = RevertFlag.DID_NOT_REVERT;
```

Comment:
I think we still need this.

Reply:
What would be the condition in which it's used?  This variable previously existed solely for the purposes of being set in `safeCREATE`, so without that logic, I'm pretty sure we're safe to remove it.